### ISSUE TITLE
several images are missing alt attributes

### ISSUE THREAD

**Issue by marlowecheng** [marlowecheng/marlowes-portfolio#18]
**Status:** Open

**Description:** Several images are missing alt attributes, which will impact accessibility rating and SEO.

**Steps to reproduce:**
1. Go to https://wave.webaim.org
2. Type in marlowecheng.com
3. View details on which images are missing alt attributes

**Expected result:** Hovering over images will produce tooltip with image description from the alt attribute in the image tag.

**Actual result:** Hovering over images did not produce tooltips. The alt attribute is empty.

**Configuration:** All platforms

**Severity:** High

**Priority:** High

**Type:** SEO

**Comment by marlowecheng** [comment]
As referenced above:
<img width="1034" alt="image" src="https://user-images.githubusercontent.com/114950923/227381388-237812f5-1dd5-4fc8-8422-7111b0d71704.png">

Result from https://www.seoptimer.com/alt-tag-checker
<img width="1094" alt="image" src="https://user-images.githubusercontent.com/114950923/227381677-0659329e-afd9-45d1-865c-0a2684dc702a.png">
